.github/workflows/main.yml

```diff
@@ -0,0 +1,64 @@
+name: vscode-bitbake CI/CD
```

Comment:
There is a merge commit in your PR that I think should be eliminated with a rebase on staging.